src/components/Drawer/Drawer.tsx

```diff
@@ -43,6 +44,7 @@ export const Drawer = ({
     <AntdDrawer
       className={styles.drawer}
       closeIcon={closeIcon}
+      data-testid={dataTestId}
```

Comment:
This prop does not exist in antd, so it does not work in theory. 
I suggest not use props if they are documented

Reply:
There are some tests, so if test passes it should work (and we can be notified if antd break this behaviour). We could trust on it?

Reply:
why risk to do a DS breaking change if antd change the behaviour? 
I suggest to use a custom node or not use for this component the data-testid and use the traditional getByX to getting the node inside your test.